frontend/src/components/Map/components/DetailsMapDrawer/index.tsx

```diff
@@ -53,7 +54,13 @@ const DetailsMapDrawer: React.FC<{
   trekFamily?: TrekFamily | null;
   trekId?: number;
 }> = ({ title, trekGeoJSON, trekFamily, trekId }) => {
-  const [open, setOpen] = useState(false);
+  const {
+    publicRuntimeConfig: {
+      map: { mobileMapPanelDefaultOpened },
+    },
+  } = getConfig();
```

Comment:
Here, you can use `getMapConfig` helper https://github.com/GeotrekCE/Geotrek-rando-v3/blob/main/frontend/src/components/Map/config.ts#L4